src/mapper.rs

```diff
@@ -1,9 +1,12 @@
 use std::collections::{BTreeMap, HashMap};
-use std::fmt::Write;
+use std::fmt::{Error as FmtError, Write};
 use std::iter::FusedIterator;
 
-use crate::mapping::{ProguardMapping, ProguardRecord};
-use crate::stacktrace::StackFrame;
+use crate::stacktrace::{self, StackFrame, Throwable};
+use crate::{
+    mapping::{ProguardMapping, ProguardRecord},
+    stacktrace::StackTrace,
+};
```

Comment:
```suggestion
use crate::stacktrace::{self, StackFrame, StackTrace, Throwable};
use crate::mapping::{ProguardMapping, ProguardRecord};
```

Reply:
Oh right, that's was probably auto-imported with rust-analyzer 😅 
I'll merge that together.